src/main/java/com/google/sps/data/Room.java

```diff
@@ -40,6 +40,12 @@ public static Room createRoom(List<Member> members, Queue<Video> videos, Queue<M
         return new Room(members, videos, messages, null);
     }
 
+    public static Room createRoom(List<Member> members){
+        Queue<Video> videoQueue = new LinkedList<Video>();
+        Queue<Message> messageQueue = new LinkedList<Message>();
+        return new Room(members,videoQueue, messageQueue, null);
```

Comment:
1. add field comment on the null value. 

2. this should be a construction function instead of factory.